packages/vats/tools/ibc-utils.js

```diff
@@ -1,11 +1,132 @@
-export const REMOTE_ADDR_RE =
```

Comment:
removing an export seems like a breaking change. or should we consider this internal?

Reply:
It was internal, and the regexp didn't actually work quite correctly.